### ISSUE TITLE
Add more promenient App Passwords instruction where needed

### ISSUE THREAD

**Issue by notpushkin** [notpushkin/Mailspring-Libre#49]
Right now, Gmail sign in looks like this:

![image](https://user-images.githubusercontent.com/1298948/106750550-0819a700-6639-11eb-818e-a299833a88fb.png)

It's easy to skip this and causes . Some ideas:

- Add brief step-by-step instructions instead of just linking to the guide
- Replace the password label with “App Password ([what is this?](https://support.google.com/accounts/answer/185833))”

This goes for all OAuth e-mail providers.

**Comment by kosssi** [comment]
In my opinion the screen "Connect an email account" is useless, instead we should see an email field. According to the provider then we will display specific information.

**Comment by notpushkin** [comment]
@kosssi Yeah, it makes sense, and I've implemented it some time ago, but this got lost after an update (#42). We should reintroduce that at some point!

**Comment by exprez135** [comment]
I agree! I left that out because I intended to look into OAuth stuff. I will look to reintroduce that :)
